src/screens/draft/index.tsx

```diff
@@ -0,0 +1,198 @@
+import React, {useState} from 'react';
+import {FlatList} from 'native-base';
+import {View, Text, SafeAreaView, TouchableOpacity, Alert} from 'react-native';
+import Input from '../../components/shared/input';
+import {useForm} from 'react-hook-form';
+import styles from './styles';
+import DatePicker from 'react-native-date-picker';
+// import format from 'date-fns/format';
+import {format, utcToZonedTime} from 'date-fns-tz';
+import Button from '../../components/shared/button';
+
+const Draft = ({route}: any) => {
+  const [openDatePicker, setOpenDatePicker] = useState(false);
+  const [openTimePicker, setOpenTimePicker] = useState(false);
+  const [date, setDate] = useState<Date>(new Date());
+  const [time, setTime] = useState<Date>(new Date());
+  const {teamA, teamB} = route.params;
+
+  const modalDatePicker = (state: boolean) => {
+    setOpenDatePicker(!state);
+  };
+  const modalTimePicker = (state: boolean) => {
+    setOpenTimePicker(!state);
+  };
+
+  const {
+    control,
+    formState: {errors},
+    setValue,
+    handleSubmit,
+  } = useForm({
+    defaultValues: {
+      field: '',
+      date: '',
+      time: '',
+    },
+    mode: 'onSubmit',
+    reValidateMode: 'onSubmit',
+  });
+
+  const averageSkillTeamA = Math.floor(
+    teamA.reduce(
+      (accumulator: number, initialValue: any) =>
+        accumulator + initialValue.skill,
+      0,
+    ) / teamA.length,
+  );
+
+  const averageSkillTeamB = Math.floor(
+    teamB.reduce(
+      (accumulator: number, initialValue: any) =>
+        accumulator + initialValue.skill,
+      0,
+    ) / teamB.length,
+  );
+
+  const listSeparator = () => <View style={styles.separator} />;
+
+  const renderItemTeamA = (data: any) => {
```

Comment:
A player interface should be added, typing data as
```suggestion
  const renderItemTeamA = (data: IPlayer) => {
```

Reply:
Done.